scripts/push-to-chocolatey/push-to-chocolatey

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+die() {
+    echo $* >&2
+    exit 1
+}
+
+if [ -z "${CHOCOLATEY_API_KEY:-}" ] ; then
+    die '$CHOCOLATEY_API_KEY is not set'
+fi
+
+apt update
+apt install unzip
```

Comment:
Maybe wrap this in an `if` so that it's skipped if unzip is there (useful when debugging: it's faster and removes the need to run the script as root):

```suggestion
if ! command -v unzip > /dev/null ; then
    apt update
    apt install unzip
fi
```